Headers/Public/VLCMediaPlayer.h

```diff
@@ -135,6 +136,19 @@ extern NSString * VLCMediaPlayerStateToString(VLCMediaPlayerState state);
  */
 - (void)mediaPlayerSnapshot:(NSNotification *)aNotification;
 
+/**
+ * Sent by the default notification center whenever the player started recording.
+ * @param player the player who started recording
+ */
+- (void)mediaPlayerRecordStarted:(VLCMediaPlayer *)player;
```

Comment:
English grammar nag nag. Shouldn't this be "mediaPlayerStartedRecording"?

Reply:
yes that would indeed sound nicer